app/src/main/java/com/bignerdranch/android/geoquiz/CheatActivity.kt

```diff
@@ -0,0 +1,64 @@
+package com.bignerdranch.android.geoquiz
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Button
+import com.bignerdranch.android.geoquiz.databinding.ActivityCheatBinding
+
+const val EXTRA_ANSWER_SHOWN =
+    "com.bignerdranch.android.geoquiz.answer_shown"
+private const val EXTRA_ANSWER_IS_TRUE =
+    "com.bignerdranch.android.geoquiz.answer_is_true"
+
+class CheatActivity : AppCompatActivity() {
+
+    private var answerIsTrue = false
+    private var binding: ActivityCheatBinding? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setupViewBinding()
+        answerIsTrue = intent.getBooleanExtra(EXTRA_ANSWER_IS_TRUE, false)
+        setupClickListener()
+    }
+
+    private fun setupClickListener() {
+        binding?.showAnswerButton?.setOnClickListener {
+            val answerText = when {
+                answerIsTrue -> R.string.true_button
+                else -> R.string.false_button
+            }
```

Comment:
I'd use a boolean condition because you have only two conditions and you cannot have more conditions